### PR TITLE
[Client-1970] node.js client version 5.1.1

### PR DESCRIPTION
**Pull request by DomPeliniAerospike**
- Improvements
  - [CLIENT-1942] Improved examples within the API documentation.
  - [CLIENT-1943] Added documentation for list, map, hll, and bit expressions.
  - [CLIENT-1944] Revamped existing API documentation.

[CLIENT-1942]: https://aerospike.atlassian.net/browse/CLIENT-1942?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CLIENT-1943]: https://aerospike.atlassian.net/browse/CLIENT-1943?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CLIENT-1944]: https://aerospike.atlassian.net/browse/CLIENT-1944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by DomPeliniAerospike** [comment]
Commits labelled CLIENT-1970 are the only new changes. All other changes were made in 5.1.0 and have been reviewed already.

File: README.md [hunk 1]
```diff
@@ -19,9 +19,16 @@ record using the Aerospike database.
 ```js
 const Aerospike = require('aerospike')
 
+// INSERT HOSTNAME AND PORT NUMBER OF AEROPSIKE SERVER NODE HERE!
 const config = {
-  hosts: '192.168.33.10:3000'
+  hosts: '192.168.33.10:3000',
```
Comment (jhecking) [inline README.md:24]: I tend to use the `AEROSPIKE_HOSTS` environment variable to set the server address(es) – I think we should at least point that option out as an alternative as well. That option is currently not very well documented, except in the [`Config#hosts` docs](https://docs.aerospike.com/apidocs/nodejs/config#hosts) and the [Getting Started tutorial](https://docs.aerospike.com/apidocs/nodejs/tutorial-getting_started.html).

Comment (DomPeliniAerospike) [reply README.md:24]: You are right. I will create a tutorial in a future release.  It is mentioned, but it could be emphasized.

**Review by artanderson — APPROVED** [review]
Just one comment, for readability I'm a much bigger fan of `async/await` with `try{}catch{}finally{}` than `then().catch()`, but that's just my personal preference.

Otherwise, looks good!

**Pull request merged**